ioservice/io_foms.c

```diff
@@ -2321,6 +2321,33 @@ static int m0_io_fom_cob_rw_tick(struct m0_fom *fom)
 	M0_ASSERT(rc == M0_FSO_AGAIN || rc == M0_FSO_WAIT);
 
 	M0_ASSERT(m0_io_fom_cob_rw_invariant(fom_obj));
+
+	if (m0_fom_phase(fom) == M0_FOPH_SUCCESS &&
+	    m0_is_write_fop(fom->fo_fop)) {
+		int                 bc_rc;
+		uint64_t            old_cob_size;
+		struct m0_cob_bckey key;
+
+		key.cbk_pfid = pver;
+		key.cbk_user_id = M0_BYTECOUNT_USER_ID;
+		bc_rc = fom_cob_locate(fom);
+		if (bc_rc == 0) {
+			cob = fom_obj->fcrw_cob;
+			old_cob_size = cob->co_nsrec.cnr_size;
+			cob_bytecount_increment(cob, &key, byte_count,
```

Comment:
Use rc here.

Reply:
Done, removed this entire section to io_finish().